src/tests/MCTS/MCTSTests.py

```diff
@@ -1,7 +1,7 @@
 import unittest
 
-from src.mcts.MCTSSolver import MCTSSolver, max_iter, max_time_s, expansion_choose_meth
-from src.tests.MCTS.NaughtCrossesState import NaughtCrossesState
+from mcts.MCTSSolver import MCTSSolver, max_iter, max_time_s, expansion_choose_meth
```

Comment:
What, this is terrible, why have you done this. Change it!!

Reply:
no